can_tools/scrapers/official/ID/id_county.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+import us
+
+from can_tools.scrapers.official.base import TableauDashboard
+from can_tools.scrapers import variables
+class IdahoCountyVaccine(TableauDashboard):
+    has_location = False
+    source = "https://coronavirus.idaho.gov/"
+    source_name = "Idaho Official Government Website"
+
```

Comment:
if we add the attribute `data_tableau_table = 'Vax Rate / County Chart'` we can use the standard `fetch()` method and save a few lines of code, (see below)
```suggestion
    data_tableau_table = 'Vax Rate / County Chart
```